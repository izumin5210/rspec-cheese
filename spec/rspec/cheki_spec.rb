require "spec_helper"

describe RSpec::Cheki do
  it "has a version number" do
    expect(RSpec::Cheki::VERSION).not_to be nil
  end

  it "says cheese!" do
    expect("foo\nbar\nbaz\n").to say_cheese
  end
end
