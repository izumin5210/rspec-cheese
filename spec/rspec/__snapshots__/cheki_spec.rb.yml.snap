---
"./spec/rspec/cheki_spec.rb[1:2]": |
  foo
  bar
  baz
